src/main/java/org/apache/maven/plugins/javadoc/JavadocReport.java

```diff
@@ -27,7 +27,7 @@
 import java.util.ResourceBundle;
 import java.util.stream.Collectors;
 
-import org.apache.maven.doxia.sink.Sink;
+import org.codehaus.doxia.sink.Sink;

```

Comment:
??
I don't think it's a right change. but looks to be back in the past :)

Reply:
> ?? I don't think it's a right change. but looks to be back in the past :)

I think that's why MSHARED-1118 has "Restore" in its title
- https://github.com/apache/maven-reporting-api/pull/13

Reply:
doh :)

Reply:
@olamy If you read the reason for the change you'll see that it was my fault to replace this in a minor version. I am working on a followup to this to make it right. Unfortunately, MJAVADOC does not/cannot use maven-reporting-impl which would make this much easier.